docs/packaging/package.yml.md

```diff
@@ -214,6 +214,15 @@ Existing Haskell packages may use the old `cabal_build`, `cabal_install`, `cabal
 | **%gem_build**   | Runs `gem build`.                                   |
 | **%gem_install** | Runs `gem install` with the appropriate parameters. |
 
+### Rust (cargo) actionable macros
+
+| Macro              | Description                                                                      |
+| ------------------ | -------------------------------------------------------------------------------- |
+| **%cargo_fetch**   | Runs `cargo fetch --locked` to get dependencies.                                 |
+| **%cargo_build**   | Runs `cargo build` with some additional flags.                                   |
+| **%cargo_install** | Installs the built binary to `/usr/bin/PACKAGE-NAME`, OR to `/usr/bin/ARGUMENT ` |
```

Comment:
If anyone can think of a better way to describe this behavior, I'm all ears.

Reply:
Show examples maybe?

For inspiration, see serpent-os/moss#202